lib/frontend/get-jira-configuration.js

```diff
@@ -25,25 +28,34 @@ const formatDate = function (date) {
 module.exports = async (req, res) => {
   const jiraHost = req.query.xdm_e;
   const { client } = res.locals;
-
   const subscriptions = await Subscription.getAllForHost(jiraHost);
-  const installations = await Promise.all(subscriptions.map(subscription => getInstallation(client, subscription)));
+  const installations = await Promise.all(subscriptions
+    .map(subscription => getInstallation(client, subscription)));
+
   const connections = installations
     .filter(response => !response.error)
     .map(data => ({
       ...data,
       isGlobalInstall: data.repository_selection === 'all',
       installedAt: formatDate(data.updated_at),
       syncState: data.syncState,
+      repoSyncState: data.repoSyncState,
     }));
+
   const failedConnections = installations.filter(response => response.error);
 
-  res.render('jira-configuration.hbs', {
-    host: req.query.xdm_e,
-    connections,
-    failedConnections,
-    hasConnections: connections.length > 0 || failedConnections.length > 0,
-    APP_URL: process.env.APP_URL,
-    csrfToken: req.csrfToken(),
-  });
+  try {
+    res.render('jira-configuration.hbs', {
+      host: jiraHost,
+      connections,
+      failedConnections,
+      hasConnections: connections.length > 0 || failedConnections.length > 0,
+      APP_URL: process.env.APP_URL,
+      csrfToken: req.csrfToken(),
+    });
+    return 'Jira configuration rendered successfully.';
+  } catch (error) {
+    console.error(error);
```

Comment:
the `req` object has a probot logger instance attached to it in https://github.com/integrations/jira/blob/master/lib/middleware/log-middleware.js. We should probably use that for the extra context/json formatting.